### ISSUE TITLE
make: *** [Makefile:46: install] Error 1

### ISSUE THREAD

**Issue by sxlmnwb** [cosmos/relayer#1408]
```
❯ cd relayer
❯ make install
installing rly binary...
main.go:4:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
cmd/tx.go:10:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
cmd/query.go:10:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
cmd/feegrant.go:7:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
cmd/keys.go:25:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
cmd/query.go:11:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
cmd/query.go:12:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/grpc_query.go:11:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/feegrant.go:10:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/tx.go:18:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/feegrant.go:11:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:4:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:5:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:6:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/query.go:16:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/grpc_query.go:12:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/tx.go:21:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/tx.go:22:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/provider.go:12:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/provider.go:13:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/tx.go:23:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/provider.go:14:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/cosmos_chain_processor.go:11:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/provider.go:15:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/query.go:19:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/keys.go:9:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/tx.go:27:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:8:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:9:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/broadcast.go:4:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/keys.go:11:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/tx.go:30:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/tx.go:31:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:11:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/grpc_query.go:15:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/account.go:10:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:12:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/feegrant.go:16:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/tx.go:35:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:13:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:14:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/account.go:11:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:15:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:16:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/query.go:24:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:17:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:18:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:19:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:20:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:21:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:22:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:23:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/query.go:25:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:24:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:25:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/query.go:26:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/keys.go:13:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/grpc_query.go:18:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:26:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:27:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:28:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/log.go:10:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:29:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/log.go:11:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/codec.go:30:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/cosmos_chain_processor.go:13:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/cosmos_chain_processor.go:14:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/provider.go:20:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/query.go:32:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/cosmos_chain_processor.go:16:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/query.go:34:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/tx.go:45:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
client/client_wrapper.go:7:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
client/client_wrapper.go:8:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
client/client_wrapper.go:9:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
client/client_wrapper.go:10:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
client/client_wrapper.go:11:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
client/client_wrapper.go:13:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
client/client_wrapper.go:14:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
client/client_wrapper.go:19:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
client/client_wrapper.go:20:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
client/client_wrapper.go:21:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
client/client_wrapper.go:22:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
client/client_wrapper.go:23:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
client/client_wrapper.go:24:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
client/client_wrapper.go:25:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
client/client_wrapper.go:26:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
client/client_wrapper.go:27:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
client/client_wrapper.go:28:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/provider/provider.go:19:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/provider/provider.go:20:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/processor/metrics.go:6:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/processor/metrics.go:7:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/processor/event_processor.go:6:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/keys/sr25519/keys.pb.go:11:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/module/app_module.go:7:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/module/app_module.go:9:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/module/app_module.go:10:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/stride/codec.go:4:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/stride/messages.pb.go:9:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/stride/codec.go:7:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/codecs/ethermint/algorithm.go:4:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/codecs/ethermint/algorithm.go:5:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/codecs/ethermint/encoding.go:13:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/codecs/ethermint/eip712.go:8:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/codecs/ethermint/algorithm.go:8:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/codecs/ethermint/encoding.go:22:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/codecs/ethermint/encoding.go:23:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/codecs/ethermint/algorithm.go:9:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/codecs/ethermint/eip712.go:13:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/codecs/ethermint/algorithm.go:6:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/codecs/ethermint/encoding.go:14:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/codecs/ethermint/encoding.go:15:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/codecs/injective/params.go:9:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/codecs/injective/params.go:10:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/codecs/injective/ethsecp256k1.go:15:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/codecs/injective/tx.pb.go:15:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/codecs/injective/tx.pb.go:16:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/codecs/injective/tx.pb.go:17:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/codecs/injective/params.go:6:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/grpc_query.go:22:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/grpc_query.go:23:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/cosmos/account.go:13:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
cregistry/cosmos_github_registry.go:12:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
cregistry/chain_info.go:16:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
internal/relaydebug/debugserver.go:10:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/penumbra/penumbra_chain_processor.go:10:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/chains/penumbra/tx.go:34:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/relayMsgs.go:10:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
relayer/path.go:12:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
cmd/appstate.go:14:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
cmd/root.go:32:2: github.com/strangelove-ventures/cometbft-client@v0.0.0-20240122193328-9503d3144af6: invalid version: unknown revision 9503d3144af6
make: *** [Makefile:46: install] Error 1
❯ go version
go version go1.21.5 linux/amd64

░▒▓    ~/relayer  on  #v2.5.1 ························································································································ with relayer@sxlmnwb  at 06:06:22 AM  ▓▒░
❯
```

**Comment by jtieri** [comment]
thanks for opening the issue, this is fixed in #1407 just waiting for it to be merged then we can tag a new release. sorry for the inconvenience!

**Comment by jtieri** [comment]
#1407 is merged and this dependency issue is resolved, i will get a new tag up asap! thanks again for opening an issue 🙂
